.github/workflows/cibuildwheel.yml

```diff
@@ -13,12 +13,13 @@ jobs:
       fail-fast: false
       matrix:
         arch: [x86_64, aarch64]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
In yaml 3.10 MUST be quoted… https://dev.to/hugovk/the-python-3-1-problem-85g

Reply:
@cclauss Thanks!